client/my-sites/stats/activity-log/index.jsx

```diff
@@ -14,14 +15,269 @@ import { getSiteSlug, isJetpackSite } from 'state/sites/selectors';
+			user: { ID: 123, name: 'Jane A', role: 'Admin' },
+			type: 'site_backed_up',
+			timestamp: 1485220539222
+		},
+		{
+			title: 'This is some really cool post',
+			description: '',
+			user: { ID: 123, name: 'Jane A', role: 'Admin' },
+			type: 'site_backed_up',
+			timestamp: 1485220539222
+		},
+		{
+			title: 'This is some really cool post',
+			description: '',
+			user: { ID: 123, name: 'Jane A', role: 'Admin' },
+			type: 'site_backed_up',
+			timestamp: 1485220539222
+		},
+		{
+			title: 'Jetpack updated to 4.5.1',
+			subTitle: 'Plugin Update',
+			description: '',
+			icon: 'plugins',
+			user: { ID: 123, name: 'Jane A', role: 'Admin' },
+			time: '4:32pm',
+			actionText: 'Undo',
+			timestamp: 1483351202400
+		},
+		{
+			title: 'Jetpack updated to 4.5.1',
+			subTitle: 'Plugin Activated',
+			description: '',
+			icon: 'plugins',
+			user: { ID: 123, name: 'Jane A', role: 'Admin' },
+			time: '4:32pm',
+			actionText: 'Undo',
+			timestamp: 1483351202420
+		},
+		{
+			title: 'Post Title',
+			subTitle: 'Post Updated',
+			description: '',
+			icon: 'posts',
+			user: { ID: 333, name: 'Jane A', role: 'Admin' },
+			time: '10:55am',
+			actionText: 'Undo',
+			timestamp: 1483264820300
+		}
+	];
+
+	update_logs( log ) {
+		const { translate } = this.props;
+		switch ( log.type ) {
+			// Scans
+			case 'security_threat_found':
+				log.subTitle = translate( 'Security Threat Found!' );
+				log.icon = 'notice';
+				log.actionText = translate( 'Fix it' );
+			// Backups
+			case 'site_backed_up':
+				log.subTitle = translate( 'Site Backed up' );
+				log.icon = 'checkmark';
+				log.status = 'is-success';
+				log.actionText = 'Restore';
+				break;
+			// Backups
+			case 'site_backed_up_failed':
+				log.subTitle = translate( 'Site Backed up' );
+				log.icon = 'notice';
+				log.actionText = 'Fix';
+				log.status = 'is-error';
+				break;
+			// Plugins
+			case 'plugin_activated':
+				log.subTitle = translate( 'Plugin Activated' );
+				log.icon = 'plugins';
+				log.actionText = 'Deactivate';
+				break;
+			case 'plugin_deactivated':
+				log.subTitle = translate( 'Plugin Deactivated' );
+				log.icon = 'plugins';
+				log.actionText = 'Activate';
+				break;
+			case 'plugin_needs_update':
+				log.subTitle = translate( 'Plugin Update Available' );
+				log.icon = 'plugins';
+				log.status = 'is-warning';
+				log.actionText = 'Update';
+				break;
+			case 'suspicious_code':
+				log.subTitle = translate( 'Security Scan' );
+				log.icon = 'notice';
+				log.status = 'is-error';
+				log.actionText = 'Fix';
+				break;
+			case 'plugin_updated':
+				log.subTitle = translate( 'Plugin Updated' );
+				log.icon = 'plugins';
+				log.actionText = 'Revert';
+				break;
+
+			// Themes
+			case 'theme_switched':
+				log.subTitle = translate( 'Theme Activated' );
+				log.icon = 'themes';
+				log.actionText = 'Revert';
+				break;
+			case 'theme_updated':
+				log.subTitle = translate( 'Theme Updated' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **8** times:
`translate( 'Green Theme Updated' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).